pkg/kubemark/hollow-kubelet.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubemark
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	kubeletapp "k8s.io/kubernetes/cmd/kubelet/app"
+	"k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/kubelet/cadvisor"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+	"k8s.io/kubernetes/pkg/kubelet/dockertools"
+	"k8s.io/kubernetes/pkg/volume/empty_dir"
+
+	"github.com/golang/glog"
+)
+
+type HollowKubelet struct {
+	KubeletConfig *kubeletapp.KubeletConfig
+}
+
+func makeTempDirOrDie(prefix string, baseDir string) string {
```

Comment:
This function has nothing to do with kubemark (in fact there is exactly the same function in cmd/integration/integration.go)
Can you please refactor it by moving this function (probably) to test/integration/utils.go